core/assignment.go

```diff
@@ -230,10 +236,18 @@ func (c *StdAssignmentCoordinator) ValidateChunkLength(state *OperatorState, blo
 // too large for the constraint in ValidateChunkLength
 func (c *StdAssignmentCoordinator) CalculateChunkLength(state *OperatorState, blobLength, targetNumChunks uint, param *SecurityParam) (uint, error) {
 
+	if targetNumChunks != 0 {
```

Comment:
Why target num chunks has to be zero? Actually it's not used below.

Reply:
I think we should deprecate this feature:
- It's currently not used.
- It's insufficient for the intended purpose (DAS)

New design here: https://www.notion.so/eigen-labs/Data-Allocation-v3-a75080bd7d334365809bbb2cfcaae538